types/config.go

```diff
@@ -128,18 +127,6 @@ func (config *Config) SetFullFundraiserPath(fullFundraiserPath string) {
 	config.fullFundraiserPath = fullFundraiserPath
 }
 
-// Set the BIP-0044 Purpose code on the config
-func (config *Config) SetPurpose(purpose uint32) {
```

Comment:
Have you checked on sourcegraph if those were used?

Reply:
So  Get/SetPurpose is redundant, since it is never really used